handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java

```diff
@@ -122,10 +122,14 @@ public boolean isRetryEnabled(String microservice) {
   }
 
   public int getRetryOnNext(String microservice) {
+    return getRetry(microservice, PROP_RETRY_ONNEXT);
+  }
+
+  private int getRetry(String microservice, String propRetry) {
```

Comment:
getRetry can change to a better name

Reply:
Name changing takes com.netflix.client.DefaultLoadBalanceRetryHandler naming as reference.